Sources/CLI/Initializer.swift

```diff
@@ -0,0 +1,56 @@
+import Foundation
+import Configuration
+
+protocol FileInteractor {
+	func contents(atPath path: String) throws -> Data
+	func fileExists(atPath path: String) -> Bool
+	func createFile(atPath path: String, withIntermediateDirectories createIntermediates: Bool, contents data: Data?, attributes attr: [FileAttributeKey : Any]?) throws
```

Comment:
<!--
  1 failure:  Colons should be ...
  0 warning: 
  
  
  DangerID: danger-id-default;
  File: Sources/CLI/Initializer.swift;
  Line: 7;
-->
- :no_entry_sign: Colons should be next to the identifier when specifying a type and next to the key in dictionary literals. (`colon`)